ccnmtldjango/template/+package+/main/tests/test_views.py

```diff
@@ -33,7 +33,7 @@ def setUp(self):
 
     def test_page(self):
         r = self.c.get("/pages/section-1/")
-        self.assertEqual(r.status_code, 200)
+        self.assertEqual(r.status_code, 403)
```

Comment:
Right - we definitely still want to allow anonymous pagetree access like Pedialabs. I'm glad these tests exposed this bug - I'll submit a PR for pagetree.